_updates/2019-12-11-december.md

```diff
@@ -0,0 +1,64 @@
+---
+title: December 2019
+---
+
+## 🎄🎁 Christmas 2019 ⛄️❄️
+
+In this release our sensenet logo has borrowed 🎅 Santa's hat to make people happy.😊  
+We also would like to make you blissful so here are the changes we made 👇🏻
+
+- We have upgrage our code to Typescript 3.7 🎉
+- Our example apps have been moved to sn-client monorepo 🏁:
+
+  [sn-react-imageupload](https://sn-react-imageupload.netlify.com)  
+  [sn-react-imagegallery](https://sn-react-imagegallery.netlify.com)  
+  [sn-react-memoapp](https://sn-react-memoapp.netlify.com)  
+  [sn-react-calendar](https://sn-react-calendar.netlify.com)  
+  [sn-react-browser](https://sn-react-browser.netlify.com)  
+  [sn-react-tasklist](https://sn-react-tasklist.netlify.com)  
+  [sn-react-usersearch](https://sn-react-usersearch.netlify.com)
+
+  Thank you for the contributors:  
+  [blaskodaniel](https://github.com/blaskodaniel),
+  [Khegyi](https://github.com/Khegyi),
+  [VargaJoe](https://github.com/VargaJoe)
+  and [B3zo0](https://github.com/B3zo0)
+
+## Admin-ui@1.4.0
+
+- We had to provide a hotfix cause there was a critical bug in admin-ui that prevented the user from uploading. 😱
+- Custom actions for CRUD have been created and some bugs have been fixed regarding this ✔️
```

Comment:
I think this is already in the previous release. Isn't it?

Reply:
I think yes :) Removed!